frontend/src/app/shared/components/datepicker/datepicker.modal.sass

```diff
@@ -8,6 +8,9 @@
   max-width: 730px
   min-height: 200px
   box-shadow: $spot-shadow-light-mid
+  // Apply a bottom margin before the banners are loaded
+  // to avoid overlapping
+  margin-bottom: 60px
```

Comment:
I don't really get the point of this.. If I remove this line, I don't see any difference. Is it a timing issue?

Reply:
The way this occurs is as follows:

1. You open the datepicker for a WP that has a precedes/follows relation
2. The datepicker modal opens briefly without banner
3. The banner information gets loaded, showing the banner

This pushes the modal to the bottom, which might result in it overlapping the window border and being cut off IF it was opened near the bottom.